.github/CONTRIBUTING.md

```diff
@@ -150,18 +159,73 @@ about how it works.
 
 ### Documentation
 
-The main documentation is written in [Markdown][] using [MkDocs][]. Dredd uses
-[ReadTheDocs][] to build and publish the documentation:
+Dredd's documentation is written in [Markdown][] using [Sphinx][]. [ReadTheDocs][] is used to build and publish the documentation:
 
 - [https://dredd.readthedocs.io](https://dredd.readthedocs.io) - preferred long URL
 - [http://dredd.rtfd.org](http://dredd.rtfd.org) - preferred short URL
 
-Source of the documentation can be found in the [docs][] directory. To contribute to Dredd's documentation, you will need to follow the [MkDocs installation instructions](http://www.mkdocs.org/#installation). Once installed, you may use following commands:
+Source of the documentation can be found in the [docs][] directory. To render Dredd's documentation on your computer, you need Python 3 and Node.js installed.
```

Comment:
I haven't try it out, but I believe it should be okay with any Python 3. RTD has 3.5, so it might make sense to pin the version to that (at least by mentioning it in docs).